stable/spark-operator/templates/rbac.yaml

```diff
@@ -25,6 +26,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: {{ include "spark-operator.fullname" . }}-crb
+  annotations:
```

Comment:
no need to declare if empty / no default values / no helm values

Reply:
Done